prow/config/inrepoconfig.go

```diff
@@ -186,6 +201,11 @@ func DefaultAndValidateProwYAML(c *Config, p *ProwYAML, identifier string) error
 		}
 	}
 
+	if len(errs) == 0 {
+		log := logrus.WithField("repo", identifier)
+		log.Debugf("Successfully got %d presubmits and %d postsubmits from %q.", len(p.Presubmits), len(p.Postsubmits), inRepoConfigFileName)
+	}
```

Comment:
Did you meant to leave this? If so please remove the part about the file name since that is not necessarily accurate.